### ISSUE TITLE
Find/searching albums

### ISSUE THREAD

**Issue by rsinghal** [ProjectMirador/mirador-design#17]
In Mirador 2.0, we made a conscious decision to call the search box a "filter" because the code is *not* searching every string in the manifest.  It is only doing a filter on some elements in the HTML.  In this case, just the title and the source.  I don't want users to think they are getting a discovery environment within the catalog find and filter area.

**Comment by ggeisler** [comment]
@rsinghal Yep, good point. I think we can just update the label to remove the term "search" and use "filter".
